packages/manager/modules/support/src/tickets/new-ticket/issues-selector/issues-selector.html

```diff
@@ -1,22 +1,21 @@
-<oui-spinner data-ng-if="$ctrl.isLoading && !$ctrl.issueTypes">
-</oui-spinner>
+<oui-field
+    data-label="{{:: 'ovhManagerSupport_new_issue' | translate }}"
+    data-size="xl">
+    <oui-spinner data-ng-if="$ctrl.isLoading"></oui-spinner>
 
-<oui-field data-label="{{ ('ovhManagerSupport_new_issue' | translate) + ' ' + $ctrl.rank }}"
-           data-ng-if="!$ctrl.isLoading && $ctrl.issueTypes.length"
-           data-size="xl">
-    <oui-select name="issue"
-                data-model="$ctrl.issueType"
-                data-items="$ctrl.issueTypes"
-                data-disabled="!$ctrl.issueTypes.length"
-                data-on-change="$ctrl.onIssueTypeSelected()"
-                data-match="label">
-    </oui-select>
+    <oui-select
+        name="issue"
+        data-ng-show="!$ctrl.isLoading"
```

Comment:
Why not a `ng-if` ?

Reply:
Don't know, it seems to work with a `ngIf` so I've changed

Reply:
Ok I get it, I'm in a `oui-field` so if I `ngIf`, then I have an error has I don't have an input inside it !

😄